dev-aws/kafka-shared-msk/pubsub/tf-kafka-config-admin.tf

```diff
@@ -0,0 +1,28 @@
+# Grant pubsub/tf-kafka-config-admin permission to all the resources
+resource "kafka_acl" "tf_kafka_config_admin_topic" {
+  resource_name       = "*"
+  resource_type       = "Topic"
+  acl_principal       = "User:CN=pubsub/tf-kafka-config-admin"
+  acl_host            = "*"
+  acl_operation       = "All"
+  acl_permission_type = "Allow"
+}
+
+resource "kafka_acl" "tf_kafka_config_admin_group" {
+  resource_name       = "*"
+  resource_type       = "Group"
+  acl_principal       = "User:CN=pubsub/tf-kafka-config-admin"
+  acl_host            = "*"
+  acl_operation       = "All"
+  acl_permission_type = "Allow"
+}
+
+resource "kafka_acl" "tf_kafka_config_admin_cluster" {
+  resource_name                = "kafka-cluster"
```

Comment:
curious: are there any other clusters?

Reply:
Actually we need the cluster permissions to be able to create ACLs: https://docs.confluent.io/platform/current/security/authorization/acls/overview.html#cluster-resource-operations
I think the kafka-cluster is the default and only cluster resource